certified-connectors/ZellisIntelligencePlatform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1063 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zellis Intelligence Platform",
+    "description": "This connectors consists various actions you can use to perform your daily tasks.",
+    "contact": {
+      "name": "Zellis Support",
+      "url": "https://zellis.service-now.com/csm",
+      "email": "ukmmprodmgmt@zellis.onmicrosoft.com"
+    },
+    "version": "1.0"
+  },
+  "host": "resourcelinkdev.co.uk",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/zip/odata/AbsenceCategories": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default"
```

Comment:
@basha-k There is no response schema defined, Have you tested the connector for operation responses? And please share a screenshot of any operation response in failed or success test cases.

Reply:
@basha-k Empty response schema are not allowed any more, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.